salt_sproxy/cli.py

```diff
@@ -33,7 +33,7 @@
 import salt.utils.stringutils
 
 try:
-    from salt.utils.file import fopen
+    from salt.utils.files import fopen
```

Comment:
I think this is due to the change in Salt 3000. I'd like to preserve backwards compatibility with older Salt versions, so I'd suggest taking out this import and move it into a (more complex) try block like:

```python
try:
    from salt.utils.files import fopen
except ImportError:
    try:
        from salt.utils.file import fopen
    except ImportError:
        from salt.utils import fopen
```